File: Indie Effects Git/Assets/IndieEffects/JS Classes/ChromaticAbberation.js
#pragma strict

@script RequireComponent(IndieEffects)
@script AddComponentMenu("Indie Effects/Chromatic Abberation")
import IndieEffects;

var fxRes : IndieEffects;
var shader : Shader;
private var chromMat : Material;
var vignette : Texture2D;

function Start () {
	fxRes = GetComponent(IndieEffects);
	chromMat = new Material(shader);
}

function OnPostRender () {
	chromMat.SetTexture("_MainTex", fxRes.RT);
	chromMat.SetTexture("_Vignette", vignette);
	FullScreenQuad(chromMat);
}

File: Indie Effects Git/Assets/IndieEffects/CSharp Classes/ChromaticAbberation.cs
using UnityEngine;

[RequireComponent(typeof(IndieEffects))]
[AddComponentMenu("Indie Effects/Chromatic Abberation C#")]
public class ChromaticAbberation : MonoBehaviour
{
    public IndieEffects fxRes;
    public Shader shader;
    private Material chromMat;
    public Texture2D vignette;

    public void Start () {
	    fxRes = GetComponent<IndieEffects>();
	    chromMat = new Material(shader);
    }

    public void OnPostRender () {
	    chromMat.SetTexture("_MainTex", fxRes.RT);
	    chromMat.SetTexture("_Vignette", vignette);
	    IndieEffects.FullScreenQuad(chromMat);
    }
}